samples: nrf9160: modem_shell: RF_MODEM_LIB_SHMEM_TX_SIZE to 22k (#9084)
nrfconnect/sdk-nrf: samples/nrf9160/modem_shell/README.rst

      PPP: started
      mosh:~$

>    Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
> 	Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
> 	You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).

Reviewer: ```suggestion
   Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
   Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
   You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).
```

> #. It has been noticed that the higher baudrates than the default 115200 are performing better with the usual use cases for PPP/dial up.
>    You can set the nRF9160 DK side uart setting for PPP by using MoSh command, for example ``ppp uartconf -b 921600``.
>    Remember to set the corresponding uart setting accordingly from PC side (in this example it is set within the ``pppd`` command later).

Reviewer: ```suggestion
   Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
   Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
   You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).
```
This is not a step, right? I see it rather as additional information between the steps.

Author: I put it as a step to get reader to stop and think here, but removed the step marking anyways now.
